emulator/emulator.py

```diff
@@ -51,12 +51,15 @@ def __init__(self, enable_middleware, debug) -> None:
         self._gps = None
         self._charger = None
 
-        self._sun_sensor_xp = SunSensor(900)
-        self._sun_sensor_xm = SunSensor(48000)
-        self._sun_sensor_yp = SunSensor(85000)
-        self._sun_sensor_ym = SunSensor(200)
-        self._sun_sensor_zp = SunSensor(12000)
-        self._sun_sensor_zm = SunSensor(5000)
+        sun_sensor_p = SunSensor(900,85000,12000)
```

Comment:
I would prefer to keep each sun sensor as its own device to make sure we are staying as true as we can be to the actual hardware experience. This won't affect us now, but it may affect things in the future